internal/cloud/cloud.go

```diff
@@ -86,7 +86,8 @@ func ensureVMForUser(vmHostname string, w io.Writer, username string, sshCmd *op
 
 func Shell(ctx context.Context, w io.Writer, projectDir string, githubUsername string) error {
 	color.New(color.FgMagenta, color.Bold).Fprint(w, "Devbox Cloud\n")
-	fmt.Fprint(w, "Remote development environments powered by Nix\n\n")
+	fmt.Fprint(w, "Remote development environments powered by Nix\n")
+	fmt.Fprint(w, "This virtual environment will exit after 8 hours of inactivity\n\n")
```

Comment:
I think we should be explicit that all data will be lost unless it's committed and pushed somewhere else. Exiting sounds like the environment might just shut down but won't be lost.

Reply:
++

Reply:
All the data is synced to local file system. Do we still have the lost data concern for the CLI scenario? (I hear you on the browser side) @gcurtis @Lagoja

Reply:
If the user disconnect before syncing completes, their local has all the changes, just the vm doesn't have the latest changes. I don't think users will lose their work tho

Reply:
But the user could make some changes on the VM and disconnect before they finish syncing back locally.